function.go

```diff
@@ -160,6 +173,12 @@ func NewSignupServer() *signupServer {
 		conversationsSid:           twilioConversationsSid,
 	})
 
+	mongoClient, dbName, err := getMongoClient()
+	if err != nil {
+		log.Printf("Could not connect to MongoDB: %v\n", err)
```

Comment:
It's probably best to not attempt to start the service if we can't connect to the database here
```suggestion
		log.Fatalf("Could not connect to MongoDB: %v\n", err)
```